apps/mobile/app/accountList/account/[id]/signAndSend/selectUtxoBubbles.tsx

```diff
@@ -313,7 +312,12 @@ export default function SelectUtxoBubbles() {
           </View>
         </GestureDetector>
       </View>
-      <SSMainLayout style={styles.absoluteSubmitContainer}>
+
```

Comment:
Let's remove this line

Reply:
Addressed here: [f329c7e](https://github.com/satsigner/satsigner/pull/44/commits/f329c7edf85350f17e08c19533ea05124b11426e)